webpack.config.js

```diff
@@ -0,0 +1,35 @@
+//@ts-check
+
+'use strict';
+
+const path = require('path');
+
+/**@type {import('webpack').Configuration}*/
+const config = {
+    target: 'node',
+    entry: './src/extension.ts',
+    output: {
+        path: path.resolve(__dirname, 'out/src'),
```

Comment:
Is there a reason not to put the Webpack output in `dist`? I wonder if that might help keep things more cleanly separated between the test build and distribution build. Then I think we could get rid of the `rimraf` stuff too, since running the tests won't interfere with building the extension package any more.

Reply:
was trying not to modify the directory configuration too much, afraid of breaking maintainer's workflow, the same reason I do this https://github.com/aioutecism/amVim-for-VSCode/pull/270#discussion_r432941686

But glad to know that you are in favor of separation between `dist` and `out`, will modify the PR!

Reply:
I've tried to make this separation [here](https://github.com/aioutecism/amVim-for-VSCode/pull/270/commits/2dd6c4af427041b8aa9f1b68cd5884eb3cfef989) but can't make it work because https://github.com/aioutecism/amVim-for-VSCode/blob/ffbead8335885e4f993ddd67b79ddca1de22167a/test/Framework/BlackBox.ts#L4
will then point to a different file (`out/src/extension`) than the one that gets loaded by the updated `main` in `package.json` resulted in a different instance of `Dispatch`

So, I decided to leave it out in `out` dir for now :bow:

Reply:
update:

I'm able to make it work by returning `getCurrentMode` from the entry point